packages/mirage-types-lwt/mirage-types-lwt.dev~mirage/opam

```diff
@@ -7,10 +7,10 @@ dev-repo:     "https://github.com/mirage/mirage.git"
 tags:         ["org:mirage" "org:xapi-project"]
 
 depends: [
-  "ocamlfind"
+  "ocamlfind" {build}
   "lwt"
   "cstruct" {>="1.4.0"}
   "io-page" {>="1.4.0"}
   "ipaddr"
-  "mirage-types" {>="2.8.0"}
+  "mirage-types" {>="3.0.0"}
```

Comment:
yeah, so this way we can just transplant these versions when we cut rcs (3.0.0~rc1)